sql-cli/sql_cli/macros/load_file_tasks.py.jinja2

```diff
@@ -0,0 +1,7 @@
+{% macro define_load_file_task(workflow_file, dag, yaml_content, yaml_file_name) -%}
+from sql_cli.operators.load_file import build_load_file_operator_content
+operator_content = build_load_file_operator_content({{yaml_content|safe}}, '{{yaml_file_name}}')
+{{ workflow_file.get_variable_name() }} = aql.load_file(
+    **operator_content
```

Comment:
This is much simpler! :raised_hands: did it work as expected? If yes, should we use a similar strategy for the `tranform_task` template?

Reply:
We would not like to use this approach as the generated DAG is inconsistent to read wrt to Python SQL aql tasks.
@feluelle and I are pairing right not to instead expand the Jinja template to make the generated task look similar to Python SDK example